forgerock-openbanking-uk-aspsp-as/forgerock-openbanking-uk-aspsp-as-gateway/forgerock-openbanking-uk-aspsp-as-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/as/api/registration/dynamic/DynamicRegistrationApiController.java

```diff
@@ -328,6 +337,26 @@
         }
     }
 
+    /**
+     * generateSSAForEIDAS This was a guess at to how OBIE were going to use eIDAS certs as part of the OB ecosystem.
+     *
+     * The assumption wasthat a QWac would somehow be associated with a specific software statement. However, this was
+     * unrealistic because QWacs are issued to an organisation.
+     *
+     * Open Banking Implementation Entity now issue OBWac certificates and these identify the Organisation and not
+     * the software statement. This means that when a TPP uses an OBWac or a full QWac certificate, they must have
+     * registered with the Open Banking (test) Directory and MUST provide and SSA as apart of the registration flow.
+     *
+     * So, this method is depricated and should no longer be used.
+     * @deprecated
+     * @param currentUser
+     * @param registrationRequestJws
+     * @param oidcRegistrationRequest
+     * @return
+     * @throws OBErrorException
+     * @throws NoSuchRDNInField
+     * @throws CertificateEncodingException
```

Comment:
Good description but can we please avoid adding empty `@param` and `@throws` in javadoc?!

You could add `@Deprecated` to the method itself.